install_multiple.yml

```diff
@@ -0,0 +1,4 @@
+import: install.yml
+
+install:
+  - "npm install @hapi/hapi@HAPI_VERSION"
```

Comment:
Can you add a newline to the end of the file.

Reply:
Yeah sure, my IDE usually does it for me but I need to update it to follow hapi's formatting rules.